src/built_in.rs

```diff
@@ -81,6 +104,53 @@ fn sum(args: Vec<Expression>) -> EvalResult<Expression> {
         })
 }
 
+fn assoc(args: Vec<Expression>) -> EvalResult<Expression> {
+    if args.len() < 3 {
+        Err(EvaluationError::WrongArity(3, args.len() as i64))
+    } else if let Expression::Map(MapType::PostEvaluation(mut map)) = args[0].clone() {
+        if let Expression::At(atom) = args[1].clone() {
+            map.insert(atom, args[2].clone());
+            Ok(Expression::Map(MapType::PostEvaluation(map)))
+        } else {
+            Err(EvaluationError::WrongType("key must be atom".parse().unwrap()))
+        }
+    } else {
+        Err(EvaluationError::WrongType("can only associate in a map".parse().unwrap()))
+    }
+}
+
+fn dissoc(args: Vec<Expression>) -> EvalResult<Expression> {
+    if args.len() < 2 {
+        Err(EvaluationError::WrongArity(2, args.len() as i64))
+    } else if let Expression::Map(MapType::PostEvaluation(mut map)) = args[0].clone() {
+        if let Expression::At(atom) = &args[1] {
+            map.remove(atom);
+            Ok(Expression::Map(MapType::PostEvaluation(map)))
+        } else {
+            Err(EvaluationError::WrongType("key must be atom".parse().unwrap()))
+        }
+    } else {
+        Err(EvaluationError::WrongType("can only dissociate in a map".parse().unwrap()))
+    }
+}
+
+fn get(args: Vec<Expression>) -> EvalResult<Expression> {
+    if args.len() < 2 {
+        Err(EvaluationError::WrongArity(2, args.len() as i64))
+    } else if let Expression::Map(MapType::PostEvaluation(map)) = args[0].clone() {
+        if let Expression::At(atom) = &args[1] {
+            match map.get(atom) {
+                Some(exp) => { Ok(exp.clone()) },
```

Comment:
https://clojure.org/guides/learn/hashed_colls#_adding_new_key_value_pairs
right below adding, there's "dissoc" and "get" too